filebeat/fileset/modules.go

```diff
@@ -440,10 +442,9 @@ func (reg *ModuleRegistry) ModuleAvailableFilesets(module string) ([]string, err
 
 // ModuleConfiguredFilesets return the list of configured filesets for the given module
 // it returns an empty list if the module doesn't exist
-func (reg *ModuleRegistry) ModuleConfiguredFilesets(module string) (list []string, err error) {
-	filesets, _ := reg.registry[module]
-	for name := range filesets {
-		list = append(list, name)
+func (reg *ModuleRegistry) ModuleConfiguredFilesets(module Module) (list []string, err error) {
```

Comment:
Why are we returning an error param in this function? What can return an error? Am I missing something?